StereoDB.Redis/Server.fs

```diff
@@ -70,12 +72,29 @@ let handleResponse clientId (line: string array) =
             match result with
             | ValueNone -> Nil
             | ValueSome value -> String value.Value
-        | [|"SET"|] -> Error "Wrong number of arguments for 'set' command"
         | [|"CLIENT"; "SETNAME"|] -> Error "Wrong number of arguments for 'SETNAME' command"
         | [|"CLIENT"; "SETNAME"; name|] -> 
             let connection = findClientConnection clientId
             connection.Name <- name
             Ok
+        | [|"INCR"|] -> Error "Wrong number of arguments for 'incr' command"
+        | [|"INCR"; key |] ->
+            let mutable result = 
+                db.WriteTransaction (fun ctx ->
+                    let records = ctx.UseTable(ctx.Schema.Records.Table)
+                    let value = records.Get key
+                    let result =
```

Comment:
You can take all this out of the transaction scope.
There is no need to keep it inside a transaction.

let result =
                        match value with
                        | ValueNone -> 1L
                        | ValueSome value -> (int64 value.Value) + 1L
                    records.Set { Id=key; Value = string result }
                    ValueSome result